content/vault/v1.20.x/content/docs/deploy/kubernetes/index.mdx

```diff
@@ -57,22 +57,32 @@ There are several ways to try Vault with Kubernetes in different environments.
 
 ### High level comparison of integrations
 
-There are currently 3 different integrations to help Kubernetes workloads seamlessly consume secrets from Vault, without the need to modify the application to interact directly with Vault. Each integration addresses slightly different use-cases. The following is a brief overview of the strengths of each integration.
+There are currently 4 different integrations to help Kubernetes workloads seamlessly consume secrets from Vault, without the need to modify the application to interact directly with Vault. Each integration addresses slightly different use-cases. The following is a brief overview of the strengths of each integration.
 
 #### Agent injector
+([docs](/vault/docs/platform/k8s/injector))
 
 - No durable secret storage outside Vault. All secrets written to disk are in ephemeral in-memory volumes.
 - No highly privileged service accounts required. All secrets are fetched with the pod's own service account without the need for any other service accounts to impersonate it.
-- More mature solution, with proven production record and advanced features like templating,
-  wider array of auth methods, etc.
+- Wider array of auth methods
 
 #### Vault Secrets Operator
+([docs](/vault/docs/platform/k8s/vso))
 
 - More native UX for app developers. Workloads can mount Kubernetes secrets without adding any Vault-specific configuration.
 - Reduced load on Vault. Secrets are synced per CRD instead of per consuming pod.
 - Better Vault secret availability. Kubernetes secrets act as a durable cluster-local cache of Vault secrets.
+- Ability to use templating through [SecretTransformations](/vault/docs/platform/k8s/vso/secret-transformation) to deliver secrets in the desired format
 
-#### Vault CSI provider
+#### Vault Secrets Operator CSI driver
+([docs](/vault/docs/platform/k8s/vso/vso-csi))
+
+- All secrets are written only to ephemeral volumes, for users who wish to avoid the locked-down API server permissions and etcd encryption requirements of Kubernetes Secrets.
```

Comment:
```suggestion
- Writes all secrets to ephemeral volumes to avoid the Kubernetes Secrets
  requirements for locked-down API server permissions and `etcd` encryption.
```
Style correction: write in active voice